src/config.ts

```diff
@@ -1,6 +1,9 @@
+import { DebugOptions } from './helpers/debugDeep';
+
 export type Config = {
   /** Default timeout, in ms, for `waitFor` and `findBy*` queries. */
   asyncUtilTimeout: number;
+  debugOptions?: Partial<DebugOptions>;
```

Comment:
RTL/DTL convention seems to be [`defaultXxx`](https://testing-library.com/docs/dom-testing-library/api-configuration#defaulthidden)
```suggestion
  defaultDebugOptions?: Partial<DebugOptions>;
```